app/assets/javascripts/host_edit.js

```diff
@@ -36,6 +36,32 @@ var nic_update_handler = function() {
   });
 };
 
+function schedulerHintFilterSelected(item){
```

Comment:
Seems more appropriate for `app/assets/javascripts/compute_resources/openstack/host_edit.js`, sorry as I know all our js is kind of tangled up